app/server/models/user.py

```diff
@@ -146,6 +146,10 @@ def phone(self):
     def phone(self, phone):
         self._phone = proccess_phone_number(phone)
 
+    # TODO [Philip]: Revisit when user bio attribute is implemented
+    def directory_listing(self):
+        return f"{self.phone} {self.bio}"
```

Comment:
bio is implemented. it's a custom_attribute

Reply:
Will add this in a few, currently working on the react bits finally got the hang of it, I think